source/arriving/setup_computer.rst

```diff
@@ -222,8 +222,18 @@ However, if you want to install scilpy, open a new session and follow these step
     module load StdEnv/2023 python/3.10 vtk
     virtualenv --no-download --clear ~/Envs/scilpy && source ~/Envs/scilpy/bin/activate
     pip install --no-index --upgrade pip
-    pip install pygit2~=1.12.0 scilpy==2.0.2
-    python -c 'import scilpy'
+    # Go where you want to clone scilpy
+    git clone https://github.com/scilus/scilpy.git
+    cd scilpy
+
+    # You can stay on the master, or choose a precise release of scilpy, for example with the 2.0.2 release:
+    git checkout tags/2.0.2 -b latest_release
+
+    # Don't forget to comment these lines in requirements.txt:
+    #dvc==3.48.*
+    #dvc-http==2.32.*
```

Comment:
in my experience ppl will often just copy and paste code snippets without reading them thoroughly. So I think I would say to comment the lines in `requirements.txt` as regular text instead of as a comment in the code snippet.

Reply:
Good point, do you prefer the new version?

Reply:
Oui mais je me demande si ça donne l'impression que le "Then, ..." s'applique seulement au choix de version de scilpy.